### PR TITLE
Testing/test framework v30

### PR DESCRIPTION
**Pull request by varticcata**
---
### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?
When updating UTF to 1.1.29 a lot of pipelines started to break. UTF 1.1.30 successfully handles the pipelines.

---
### Testing status
Describe what manual/automated tests were performed for this PR

HDRP_BatchRenderGroup for Linux_Vulkan_Standalone_mono_Linear on trunk - https://unity-ci.cds.internal.unity3d.com/job/11454173
ShaderGraph for Win_Dx11_Editmode_mono_Linear on trunk - https://unity-ci.cds.internal.unity3d.com/job/11454184
VFX_URP on Win_Dx11_Standalone_mono_Linear on trunk - https://unity-ci.cds.internal.unity3d.com/job/11482835
URP_2D on Linux_Vulkan_Standalone_mono_Linear on trunk - https://unity-ci.cds.internal.unity3d.com/job/11454893
LightMapping on Linux_OpenGL_Editmode_mono_Linear on trunk - https://unity-ci.cds.internal.unity3d.com/job/11455925
PostProcessing on Win_Dx11_Standalone_mono_Linear on trunk - https://unity-ci.cds.internal.unity3d.com/job/11457006
BatchRenderGroup on trunk - https://unity-ci.cds.internal.unity3d.com/job/11482878

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: TestProjects/HDRP_RuntimeTests/Packages/manifest.json [hunk 1]
```diff
@@ -1,57 +1,57 @@
...
+        "com.unity.ide.visualstudio": "2.0.0",
+        "com.unity.ide.vscode": "1.1.3",
+        "com.unity.render-pipelines.core": "file:../../../com.unity.render-pipelines.core",
+        "com.unity.render-pipelines.high-definition": "file:../../../com.unity.render-pipelines.high-definition",
+        "com.unity.render-pipelines.high-definition-config": "file:../../../com.unity.render-pipelines.high-definition-config",
+        "com.unity.shadergraph": "file:../../../com.unity.shadergraph",
+        "com.unity.test-framework": "1.1.30",
+        "com.unity.test-framework.build": "0.0.1-preview.14",
+        "com.unity.test-framework.utp-reporter": "1.0.2-preview",
+        "com.unity.testframework.graphics": "7.8.17-exp.1",
```
Comment (sophiaaar) [inline TestProjects/HDRP_RuntimeTests/Packages/manifest.json:12]: is this intended?

File: TestProjects/VisualEffectGraph_HDRP/Packages/manifest.json [hunk 2]
```diff
@@ -1,60 +1,60 @@
...
-    "com.unity.render-pipelines.visualeffectgraph",
-    "com.unity.testframework.graphics",
-    "com.unity.testing.graphics-performance"
-  ]
-}
+    "disableProjectUpdate": true,
+    "dependencies": {
+        "com.unity.ide.visualstudio": "2.0.7",
+        "com.unity.ide.vscode": "1.1.3",
+        "com.unity.inputsystem": "1.0.2",
```
Comment (theo-pnv) [inline TestProjects/VisualEffectGraph_HDRP/Packages/manifest.json:6]: It seems that some packages were downgraded, is that intended?

**Review by theo-pnv — COMMENTED** [review]
Thank you! If everything works with 1.1.30 it's great news, we've been blocked on utf 1.1.18 for ages :) 
2 things I noticed:
- I think some config files were missed when updating, for example the perf config (it lives in the yamato files: [lostcrypt](https://github.com/Unity-Technologies/Graphics/blob/testing/test-framework-v30/.yamato/config/universal_perf_lostcrypt.metafile#L38) or [vfx](https://github.com/Unity-Technologies/Graphics/blob/testing/test-framework-v30/.yamato/config/vv_perf_urp.metafile#L37), among others). I usually use the `search entire project` feature of my IDE for such tasks, for example Ctrl-Shift-F if you use VSCode. And then look for all `1.1.18` or `1.1.29` results.
- I am a little bit paranoid when it comes to updating the test framework, given I handled the last update and every project started to break... Would it make sense to run a whole nightly on the PR and also some consoles jobs to be extra sure?

**Review by sophiaaar — APPROVED** [review]

File: TestProjects/VisualEffectGraph_HDRP/Packages/manifest.json (hunk 2 above)
Comment (sophiaaar) [reply TestProjects/VisualEffectGraph_HDRP/Packages/manifest.json]: dont worry, the perf configs already have UTF at 1.1.30 :)

**Pull request merged**